airframe-http-finagle/src/main/scala/wvlet/airframe/http/finagle/package.scala

```diff
@@ -179,7 +179,12 @@ package object finagle {
   }
 
   def convertToFinagleRequest(request: HttpMessage.Request): Request = {
-    val req = http.Request()
+    val req = if (request.method == "GET") {
+      val params = request.query.toSeq.map((e) => (e.key, e.value))
```

Comment:
```suggestion
      val params = request.query.toSeq.map(e => (e.key, e.value))
```
If no type is specified, no need to wrap the arg with paren